internal/service/privatelinkendpointservicedatafederationonlinearchive/data_source_privatelink_endpoint_service_data_federation_online_archives.go

```diff
@@ -43,6 +43,14 @@ func PluralDataSource() *schema.Resource {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
+						"region": {
+							Type:     schema.TypeString,
+							Optional: true,
```

Comment:
if these cannot be set by the user, these should be `Computed`

Reply:
Fixed in 1a0ba3a and 80c0762